src/sentry/tasks/backfill_seer_grouping_records.py

```diff
@@ -458,13 +482,22 @@ def lookup_group_data_stacktrace_bulk(
                     bulk_event_ids.add(event_id)
                 else:
                     invalid_event_ids.add(event_id)
+                    invalid_event_group_ids.add(group_id)
 
     metrics.gauge(
         f"{BACKFILL_NAME}._lookup_event_bulk.hit_ratio",
         round(len(bulk_data.items()) / len(rows)) * 100,
         sample_rate=1.0,
     )
 
+    logger.info(
+        "backfill_seer_grouping_records.invalid_event_ids",
+        extra={
+            "project_id": project_id,
+            "invalid_event_group_ids": json.dumps(invalid_event_group_ids),
+        },
+    )
```

Comment:
this log name is confusing as the name is invalid_event_ids, but here we're logging the group_ids, but can follow up